Add ClearML Logger (#12)
explosion/spacy-loggers: README.md

| `tags`                 | `Optional[Dict[str, Any]]` | A dictionary of string keys and values to set as tags on the run. If a run is being resumed, these tags are set on the resumed run. If a new run is being created, these tags are set on the new run (default: `None`). |
| `remove_config_values` | `List[str]`                | A list of values to exclude from the config before it is uploaded to MLflow (default: `[]`).                                                                                                                            |

## ClearMLLogger

### Installation

This logger requires `clearml` to be installed and configured:

```bash
pip install clearml
clearml-init
```

### Usage

`spacy.ClearMLLogger.v1` is a logger that tracks the results of each training step
using the [ClearML](https://www.clear.ml/) tool. To use
this logger, ClearML should be installed and you should have initialized (using the command above).
The logger will send all the gathered information to your ClearML server, either [the hosted free tier](https://app.clear.ml) 

> or the open source [self-hosted server](https://github.com/allegroai/clearml-server). This logger captures all the following information (all visible in the webUI):

Reviewer: ```suggestion
or the open source [self-hosted server](https://github.com/allegroai/clearml-server). This logger captures all the following information (all visible in the web UI):
```

> - The full spaCy config file contents

Reviewer 2: ```suggestion
- The full spaCy config file contents.
```

- Code information such as git repository, commit ID and uncommitted changes.

> - Full console output

Reviewer 2: ```suggestion
- Full console output.
```

> - Miscellanious info such as time, python version and hardware information

Reviewer: ```suggestion
- Miscellaneous info such as time, python version and hardware information
```

> - Miscellaneous info such as time, python version and hardware information

Reviewer 2: ```suggestion
- Miscellaneous info such as time, python version and hardware information.
```

- Output scalars:
    - The final score is logged under the scalar `score`.

>     - Individual component scores are grouped together on 1 scalar plot, use the webUI to filter.

Reviewer: ```suggestion
    - Individual component scores are grouped together on one scalar plot, use the web UI to filter.
```

>     - Individual component scores are grouped together on one scalar plot, use the web UI to filter.

Reviewer 2: ```suggestion
    - Individual component scores are grouped together on one scalar plot (filterable using the web UI).
```

    - Loss values of different components are logged with the `loss_` prefix.

> Then optionally, depending on the setting below, the following can be captured:

Reviewer 2: ```suggestion
In addition to the above, the following artifacts can also be optionally captured:
```
